### ISSUE TITLE
MultipleContextManagerAssignmentsViolation with several managers

### ISSUE THREAD

**Issue by AlwxSin** [wemake-services/wemake-python-styleguide#1000]
# Bug report

## What's wrong
This rule was created to forbid `with open('') as first, second:` where it's hard to find from where was `second` variable.

But if I want to use several context managers like that
```python
with A() as a, B() as b:
    ...
```
I receive MultipleContextManagerAssignmentsViolation.

## How is that should be
MultipleContextManagerAssignmentsViolation should not raise on multiple context managers

**Comment by sobolevn** [comment]
What is the difference between `with open('') as first, second:` and `with A() as a, B() as b:`?
AFAIK, these two have the same semantics.

**Comment by AlwxSin** [comment]
In `with open('') as first, second:` we can't clearly say is `second` comes from `open()` or from something above.
In `with First() as first, Second() as second:` we can see what where comes from.

**Comment by sobolevn** [comment]
Ok, fair enough! Do you want to fix it?

**Comment by AlwxSin** [comment]
@sobolevn If you can help me and point me where should I look, I will fix it. Thanks.

**Comment by sobolevn** [comment]
Code: https://github.com/wemake-services/wemake-python-styleguide/blob/e84e7671aad08a5acc87646f1c86f20cd65ce400/wemake_python_styleguide/visitors/ast/keywords.py#L201
Docs: https://github.com/wemake-services/wemake-python-styleguide/blob/c34a3746c9d9d2879d7914d9230a2215f9ffe40d/wemake_python_styleguide/violations/consistency.py#L661
Tests: https://github.com/wemake-services/wemake-python-styleguide/blob/5591bafa19d9e8be8edc74c7f3320d8122deeca7/tests/test_visitors/test_ast/test_keywords/test_context_managers/test_context_manager_targets.py

**Comment by sobolevn** [comment]
Ok, on the other hand we have several new problems here:

1. You can now write `with open() as foo, open() as bar:` and you can also write two `with` cases. That's inconsistent.
2. It might be unclear what executes in which order. Is `foo` the first context? Or the second?
3. We will need to write extra complexity rules for `with` in this case. Currently we just decline all code with more than one target. 

All in all, I guess this is intentional. We even have a special test case for this: https://github.com/wemake-services/wemake-python-styleguide/blob/5591bafa19d9e8be8edc74c7f3320d8122deeca7/tests/test_visitors/test_ast/test_keywords/test_context_managers/test_context_manager_targets.py#L18-L22

Solution: use nested `with` cases.
